src/client/cli/cmd/mount.cpp

```diff
@@ -35,6 +36,8 @@ namespace cmd = multipass::cmd;
 namespace
 {
 constexpr auto category = "mount cmd";
+const QString default_mount_type{"integrated"};
+const QString native_mount_type{"native"};
```

Comment:
Hey @luis4a0, @ricab, @andrei-toterman, and @sharder996,

What do you guys think of these names for the type passed into `mount`?  Since this is user facing, I'd like a consensus on this before it's committed.

Reply:
Hey @townsend2010!

I'd go for something a bit different. `integrated` does not give for me the idea that it's the default in Multipass, I think `default` would be OK to confer to users the idea of method which was always there in Multipass.

OTOH, `native` seems good to me. Maybe `optimized` or `platform` would be a bit more clear.

Reply:
Hey @luis4a0,

The notion of what is "default" may change as the native mounts mature and we want to make those default, so I don't think `default` should be a user facing type.  I think it should be a simple, descriptive name for each type although I'm not quite sure `integrated` conveys that, but I can't think of a term that does.  I'd like to avoid calling out what the underlying technology is too, so I'd like to avoid things such as `sshfs`, `smb`, etc.

Reply:
I agree with both of you. "Default" may change, but `integrated` sounds a little opaque to me too. How about `portable` and `optimized`? Those would convey the main advantage of each option.

Reply:
++`portable`!

Reply:
(well, the exclamation mark above is an exclamation mark, not a factorial sign)

Reply:
Generally, I think the original names are fine as they describe the technology and not a relative attribute of it. For that reason I would shy away from using `optimized` because `optimized` could have a different meaning depending on use cause or additional future options.

Reply:
I tend to agree that `optimized` might be a little misleading, since, at least to me, it gives the impression that the underlying technology is the same, but it just has a better implementation. 
Since the end goal would be to hopefully make the performance mounts the default choice, how about:
for `sshfs`: `legacy`, since it's an older system, but it can still be used to get the job done. other alternatives: `primitive`, `basic`, `inherent`, `builtin`, `inbuilt`
for performance mounts: `native` and `platform` sound good, but `platform` is not an adjective, so it sounds kinda weird. alternatives: `express`, `specialized`, `specific`, `modern`, `targeted`

Reply:
After some discussion on the daily sync, we settled on `classic` and `native` for the mount types.